spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/RDDFunctions.scala

```diff
@@ -159,14 +158,39 @@ class RDDFunctions[T](rdd: RDD[T]) extends WritableToCassandra[T] with Serializa
   implicit
     connector: CassandraConnector = CassandraConnector(sparkContext.getConf),
     currentType: ClassTag[T],
-    rwf: RowWriterFactory[T]) = {
+    rwf: RowWriterFactory[T]): CassandraPartitionedRDD[T] = {
+
+    val converter = ReplicaMapper[T](connector, keyspaceName, tableName, partitionKeyMapper)
```

Comment:
converter? What does it convert? The class name suggests it is a replica mapper (even then, still "mapper" is overloaded, for we also use mappers for mapping classes to cassandra schema).